### ISSUE TITLE
Use different method to get key label in the no modifiers case

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/node-native-keymap#1]
Currently, for German (Swiss) the value for VKEY_OEM_5 is:
```
  { key_code: 'VKEY_OEM_5',
    value: '´ä',
    withShift: 'à',
    withAltGr: '{',
    withShiftAltGr: '' },
```

Try to make it `ä` instead of `´ä`

**Comment by alexandrudima** [comment]
See Microsoft/vscode#1358

**Comment by the-ress** [comment]
The issue is actually caused by calling `ToUnicode` for `AltGr+VKEY_OEM_4` before calling it for `VKEY_OEM_5`. The first call inserts the dead character into the buffer and then the call for `VKEY_OEM_5` returns both characters.

As described in [ToUnicodeEx documentation](https://msdn.microsoft.com/en-us/library/windows/desktop/ms646322(v=vs.85).aspx): 
> As ToUnicodeEx translates the virtual-key code, it also changes the state of the kernel-mode keyboard buffer. This state-change affects dead keys, ligatures, alt+numpad key entry, and so on.

The fix in 966f8cf only defers the issue to the next character:
```
  { key_code: 'VKEY_OEM_5',
    value: 'ä',
    withShift: '´à',
    withAltGr: '{',
    withShiftAltGr: '' },
```
